paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -119,4 +123,8 @@ class PaparazziPlugin : Plugin<Project> {
       return this
     }
   }
+
+  internal companion object {
+    const val VERIFICATION_GROUP = "verification"
```

Comment:
This is available at `org.gradle.language.base.plugins.LifecycleBasePlugin.VERIFICATION_GROUP`.